pages/boac/boac_filtered_cohort_page_filters.rb

```diff
@@ -9,7 +9,7 @@ module BOACFilteredCohortPageFilters
   # NEW FILTERED COHORTS
 
   button(:new_filter_button, xpath: '//button[starts-with(@id, \'new-filter-button\')]')
-  button(:new_sub_filter_button, xpath: '//div[contains(@id,"filter-row-dropdown-secondary")]//button')
+  button(:new_sub_filter_button, xpath: id: '//button[starts-with(@id, \'filter-row-dropdown-secondary\')]')
```

Comment:
@johncrossman I think you've got two locator types on this.  It should just be xpath, right?

Reply:
Fixing